docs/source/standard.md

```diff
@@ -113,65 +113,130 @@ The ability of the TRE organisation to provide and manage devices, workspaces, i
+  - This may be in the form of a virtual Windows or Linux desktops, web applications, or a terminal.
+    The use of custom developed TRE-specific software should be avoided when widely used open-source alternatives already exist.
+* - A TRE should take accessibility for users with disabilities into account.
+  - The restricted nature of TREs means many assistive tools such as screenreaders in a virtual desktop may not be allowed, but other options such as colour schemes, font sizes, and resizing user interface elements, should be supported.
+* - Copying out data via the system clipboard must be disabled.
+  - A TRE user must not be able to copy sensitive data out of a workspace using the system clipboard.
+    A TRE may allow user to paste text into a workspace.
+```
 
 #### 2.1.2 Software tools
 
+The tools used by researchers inside a TRE
 programming languages, IDEs, desktop applications etc.
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
-
-#### 2.1.3 High performance computing
-
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+```{list-table}
+:header-rows: 1
+:name: tab-end-user-software-tools
+* - Statement
+  - Guidance
+* - A TRE must provide software applications that are relevant to working with the data in the TRE.
+  - The tools provided will depend on the types of data in the TRE, and the expectations of users of the TRE.
+    This may include programming languages such as Python and R, integrated development environments, Jupyter notebooks, office type applications such as word processors and spreadsheets, command line tools, etc.
+    The set of tools should be reviewed regularly to ensure they are up to date.
+* - A TRE should provide tools to encourage best-practice in reproducibly analysing data.
+  - Reproducibility of analyses improves auditability and accountability of how data has been used, as well as being best-practice in research.
+    This may include version control software, and tools for developing and running data analysis pipelines.
+* - A TRE may provide shared services that are accessible to users in the same project.
+  - This may include shared file storage, databases, collaborative writing, and other web applications.
+    This must only be shared amongst users within the same project.
+* - A TRE may provide limited access to some software repositories
+  - For example, a TRE may allow installation of packages from Python or R repositories, or provide an internal mirror with approved packages.
+```
+
+#### 2.1.3 High performance or cluster computing
+
+The ability to run analyses requiring more compute resources than is present in the user's workspace.
+
+```{list-table}
+:header-rows: 1
+:name: tab-end-user-high-performance-cluster-computing
+* - Statement
+  - Guidance
+* - A TRE should provide access to high performance computing or other scaleable compute resource if required by users.
```

Comment:
I think when we say HPC here we mean a few things,

- High performance hardware (Xeon Platinum, EPYC, fibre optic networking, data centre GPUs and interconnects, ...)
- Some sort of scheduling system (SLURM, PBS, Grid Engine, HTCondor)
- High availability (either through dedicated machines or scaling) of compute resources

Do we want to talk about these things separately? Should was say that for our purposes that is what HPC means (or define our own term?).

I worry it might be confusing/misleading to other who strongly associate HPC either with hardware or HPC as a paradigm in constrast to HTC.